packages/synapse-react-client/test/lib/containers/widgets/LoginAwareButton.test.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react'
+import { renderHook, screen, waitFor } from '@testing-library/react'
+import {
+  LoginAwareButton,
+  LoginAwareButtonProps,
+} from '../../../../src/lib/containers/widgets/LoginAwareButton'
+import { SRC_SIGN_IN_CLASS } from '../../../../src/lib/utils/SynapseConstants'
+import { useApplicationSessionContext } from '../../../../src/lib/utils/apputils/session/ApplicationSessionContext'
+import { MemoryRouter } from 'react-router-dom'
+import ApplicationSessionManager from '../../../../src/lib/utils/apputils/session/ApplicationSessionManager'
+import { SynapseClient } from '../../../../src/lib'
+import { MOCK_ACCESS_TOKEN } from '../../../../mocks/MockSynapseContext'
+
+const defaultProps: LoginAwareButtonProps = {
+  variant: 'contained',
+  color: 'secondary',
+  sx: {
+    color: 'white',
+    fontWeight: 400,
+    ':hover': { color: 'white' },
+    ':active': { color: 'white' },
+    ':visited': { color: 'white' },
+  },
```

Comment:
Hopefully [this change in Palettes.ts](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/226/files#r1192714592) makes hard-coding the text color unnecessary

Reply:
I removed the styling from this test because it was superfluous. I've confirmed white is now the default in effect for button variant="contained". For variant="outlined" specifying white appears to still be necessary however:
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/824229/8d5ad31b-7881-4d9f-9ce9-8c993a3ec54c)